dMasterServer/MasterServer.cpp

```diff
@@ -144,7 +144,8 @@ int main(int argc, char** argv) {
 	MigrationRunner::RunMigrations();
 
 	// Check CDClient exists
-	if (!std::filesystem::exists(Game::assetManager->GetResPath() / "CDServer.sqlite")) {
+	if (!std::filesystem::exists(Game::assetManager->GetResPath() / "CDServer.sqlite") && 
```

Comment:
might make sense to extract those exists calls into variables if you use them multiple times.